gradle.properties

```diff
@@ -14,6 +18,12 @@ neoform_version=20241023.131943
 # on snapshot versions, used to prefix the version
 neoforge_snapshot_next_stable=21.4
 
+jst_version=1.0.45
+legacyinstaller_version=3.0.27
```

Comment:
It's at 33 already. But speaking of that, I think it's beneficial if we keep it at a patch wildcard as this way bug fixes and translations get propagated down to other branches too.

Reply:
I don't think we should use wildcards ever. Maybe @shartte has a different opinion.

Reply:
Dynamic versions here we go!